zuul.d/jobs.yaml

```diff
@@ -1099,3 +1099,23 @@
     override-checkout: branch-2.4
     vars:
       k8s_log_dir: '{{ ansible_user_dir }}/workspace/logs/kubernetes'
+
+- job:
+    name: helm-integration-test-kubeadm-k8s
```

Comment:
I think we should test fixed k8s release to avoid `latest` version changing impact test result of job changing. For example `helm-integration-test-kubeadm-k8s-v1.13.4`